source/crud/tests/unified/bulkWrite-deleteMany-hint-unacknowledged.yml

```diff
@@ -0,0 +1,97 @@
+description: bulkWrite-deleteMany-hint-unacknowledged
+
+schemaVersion: '1.0'
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents:
+        - commandStartedEvent
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name db0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+      collectionOptions:
+        writeConcern: { w: 0 }
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents: &documents
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
+tests:
+  - description: "Unacknowledged deleteMany with hint string fails with client-side error on pre-4.4 server"
+    runOnRequirements:
+      - maxServerVersion: "4.2.99"
+    operations:
+      - object: *collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - deleteMany:
+                filter: &filter { _id: { $gt: 1 } }
+                hint: _id_
+        expectError:
+          isClientError: true
+    expectEvents: &noEvents
+      - client: *client0
+        events: []
+    outcome: &outcomeUnchanged
+      - collectionName: *collection0Name
+        databaseName: *database0Name
+        documents: *documents
+
+  - description: "Unacknowledged deleteMany with hint document fails with client-side error on pre-4.4 server"
+    runOnRequirements:
+      - maxServerVersion: "4.2.99"
+    operations:
+      - object: *collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - deleteMany:
+                filter: *filter
+                hint: { _id: 1 }
+        expectError:
+          isClientError: true
+    expectEvents: *noEvents
+    outcome: *outcomeUnchanged
+
+  - description: "Unacknowledged deleteMany with hint string on 4.4+ server"
+    runOnRequirements:
+      - minServerVersion: "4.4.0"
+    operations:
+      - object: *collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - deleteMany:
+                filter: *filter
+                hint: _id_
+        expectResult: &result { $$unsetOrMatches: { acknowledged: { $$unsetOrMatches: false } } }
+    outcome: &outcomeChanged
+      - collectionName: *collection0Name
+        databaseName: *database0Name
+        documents:
+          - { _id: 1, x: 11 }
```

Comment:
It looks like the subsequent acknowledged insert was not enough to ensure both writes were applied before `outcome` is evaluated. https://github.com/mongodb/mongo-php-library/pull/859's latest CI build reported a failure for a test within `bulkWrite-deleteMany-hint-unacknowledged` on a 4.4 sharded cluster because the `outcome` query returned too many documents (i.e. deleteMany did not complete).

I think the best course of action is to keep the APM assertions (assert that commands are issued as we expect, with no client-side error) and remove the `outcome` assertions altogether, but I'll wait for some feedback on this before doing so.

Reply:
@benjirewis @alcaeus: Thoughts on removing the `outcome` assertions altogether?

Reply:
Removing the `outcome` assertions altogether seems fine to me if the aforementioned change-stream-block is annoyingly complex. If you do so, could you also remove the subsequent, acknowledged write (`insertOne`)?

Reply:
Yes, I would remove the insert as well (see: https://github.com/mongodb/specifications/pull/1057#discussion_r697625855).